src/main/java/mate/academy/hibernate/relations/model/Movie.java

```diff
@@ -2,10 +2,22 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
 
+@Entity
+@Table(name = "movies")
 public class Movie implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String title;
+    @OneToMany(fetch = FetchType.EAGER)
```

Comment:
excellent work) just a few comments:
to my mind one film can have a lot of actors and one actor can acting in a planty of films, but it's ok to use one to many relations to) just a small recomendation: you shoul use @jointable annotation to set consistent names for your columns, for example @jointable(name = "movies_actors",
joinColumns = @joincolumn(name = "movie_id"),
inverseJoinColumns = @joincolumn(name = "actor_id"))